requirements/AUR_RT-0050/Taste-Leon3-Runtime-Uart-driver.txt

```diff
@@ -1,2 +1,4 @@
-Requirement fulfilled, document reference:
-    AUR-N7S-RP-0001 D5_2 CBI Technical Architecture (5.1 Software design overview)
\ No newline at end of file
+TASTE Leon3 Runtime uses Leon3-BSP for implementing UART communication, as per [1]. 
```

Comment:
Chapter 5.1 does not mention Leon3 -BSP directly.  Let's write "uses Uart module for accessing Leon3 hardware UART, as per chapter 5.1 of [1]. Uart module is part of Leon3-BSP, as per chapter 5.2 of [1]". And modify [1] to not include chapter reference.

Reply:
Done